components/constellation/constellation.rs

```diff
@@ -1495,6 +1490,7 @@ impl<Message, LTF, STF> Constellation<Message, LTF, STF>
         };
 
         // Create the new pipeline, attached to the parent and push to pending changes
+        self.handle_load_start_msg(load_info.info.new_pipeline_id);
```

Comment:
If we need to be calling `self.handle_load_start_msg` before we push a pending change, it might make sense to add a new method for adding pending changes that way we don't forget to call `self.handle_load_start_msg`.

This could be saved for another PR since it isn't directly related to these changes. Possible E-Easy.